ktor-http/common/src/io/ktor/http/URLBuilder.kt

```diff
@@ -189,3 +189,30 @@ public val URLBuilder.authority: String
             append(port.toString())
         }
     }
+
+
+/**
+ * Adds [parts] to current [encodedPath]
+ */
+public fun URLBuilder.pathPart(parts: List<String>): URLBuilder {
```

Comment:
Path component? Path segment? Not sure about "part" 

Also consider reflecting muptiple components in the function name